File: Packets/src/main/java/me/ImSpooks/core/packets/collection/network/PacketResponseExpired.java
package me.ImSpooks.core.packets.collection.network;

import me.ImSpooks.core.packets.collection.GlobalPacket;
import me.ImSpooks.core.packets.init.Packet;
import me.ImSpooks.core.packets.init.channels.WrappedInputStream;
import me.ImSpooks.core.packets.init.channels.WrappedOutputStream;

import java.io.IOException;

/**
 * Created by Nick on 13 Oct 2019.
 * Copyright © ImSpooks
 */
@GlobalPacket
public class PacketResponseExpired extends Packet {

    private long ms;

    public PacketResponseExpired(long ms) {
        this.ms = ms;
    }

    public PacketResponseExpired() {
    }

    @Override
    public void send(WrappedOutputStream out) throws IOException {
        out.writeLong(this.ms);
    }

    @Override
    public void receive(WrappedInputStream in) throws IOException {
        this.ms = in.readLong();
    }
}

File: ClientBridge/src/main/php/client/packets/collection/network/PacketResponseExpired.php
<?php
/**
 * Created by Nick on 24 okt 2019.
 * Copyright © ImSpooks
 */

namespace client\packets\collection\network;

use client\packets\init\Packet;
use client\packets\init\channels\WrappedInputStream;
use client\packets\init\channels\WrappedOutputStream;

class PacketResponseExpired extends Packet {

    /** @var int */ 	private $ms;

    public function __construct(int $ms = 0) {
        $this->ms = $ms;
    }

    public function receive(WrappedInputStream $in) {
        $this->ms = $in->readLong();
    }

    public function send(WrappedOutputStream $out) {
        $out->writeLong($this->ms);
    }

}
